src/reducers-actions/erc20.ts

```diff
@@ -0,0 +1,50 @@
+import { TokenErc20 } from '@gnosis.pm/dex-js'
```

Comment:
I'd love to follow the structure of uni, I like how they separate have all slices of the state in folders, and they split actions, hooks, redicers ...

Reply:
That's on this PR :) https://github.com/gnosis/gp-ui/pull/129